pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -151,7 +165,19 @@ async def initialize(self):
         await self._initialize_context()
 
         # indicate that we're ready
-        await self._write_packet_to_processor(self._processor_sock, 's')
+        await self._write_packet_to_processor(self._event_sock, 's')
+        await self._send_data_on_control_socket({
+            'kind': 'wrapperInitialized',
+            'attributes': {'ready': 'true'}
+        })
+
+    async def receive_control_messages(self):
+
+        control_message_event_length = await self._resolve_event_message_length(self._control_sock)
+
+        control_message_event = await self._resolve_event(self._control_sock, control_message_event_length)
+
+        self._logger.info_with('Received control message', contorl_message=control_message_event.body)
```

Comment:
```suggestion
        self._logger.debug_with('Received control message', contorl_message=control_message_event.body)
```

Reply:
every log >= info means cpu time on production-like functions

Reply:
Good to know 👍🏻